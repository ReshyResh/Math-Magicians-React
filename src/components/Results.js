import PropTypes from 'prop-types';

const Results = ({ value }) => <p className="result">{value}</p>;

Results.propTypes = {
  value: PropTypes.string,
};

Results.defaultProps = {
  value: '0',
};

export default Results;
