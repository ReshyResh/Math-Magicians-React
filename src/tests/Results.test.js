import React from 'react';
import renderer from 'react-test-renderer';
import Results from '../components/Results';

it('Results renders correctly', () => {
  const tree = renderer
    .create(<Results />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
