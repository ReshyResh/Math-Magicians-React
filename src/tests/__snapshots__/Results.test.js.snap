// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results renders correctly 1`] = `
<p
  className="result"
>
  0
</p>
`;
